solara/website/components/contact.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+import requests
+
+from typing import Any, Dict, Optional
+import solara
+
+
+postmark_api_key = None
+contact_email_address = None
+
+try:
+    postmark_api_key = os.environ["POSTMARK_API_KEY"]
+    contact_email_address = os.environ["SOLARA_CONTACT_EMAIL_ADDRESS"]
+except Exception:
+    pass
+
+
+@solara.component
+def Contact(
+    style: Dict[str, Any] = {},
+    title="Contact Us",
+    subtitle="We'd love to hear from you!",
+    submit_label="Submit",
+    email_subject="Contact Form Submission",
+):
+    first_name = solara.use_reactive("")
+    last_name = solara.use_reactive("")
+    email = solara.use_reactive("")
+    company = solara.use_reactive("")
+    message = solara.use_reactive("")
+    error: solara.Reactive[Optional[str]] = solara.use_reactive(None)
+
+    def send(*_ignore):
```

Comment:
I think i would have done this with a task, and also do the error handling with that. Ok to keep it as it is for now.

Reply:
I didn't notice the operation taking any time, so I didn't even come to think of using task. It would probably be a more robust implementation, so maybe we can implement it later